neuralpredictors/layers/cores/conv2d.py

```diff
@@ -146,10 +146,14 @@ def __init__(
         self.linear = linear
 
         if depth_separable:
+            self.conv_layer_name = "ds_conv"
             self.ConvLayer = DepthSeparableConv2d
         elif attention_conv:
+            # TODO: check if name attention_conv is backwards compatible
+            self.conv_layer_name = "attention_conv"
```

Comment:
So far, the attention conv was only used by me in the SECore2d as far as I know. There, I didn't give the layer a specific name, but just `conv`. 
So to make it backwards compatible, I think it would be best to also just name it `conv`.

https://github.com/sinzlab/nnvision/blob/c9e69c308248f50c4362254eae68f70414570076/nnvision/models/cores.py#L200

Reply:
We agreed on keeping the more descriptive names because the module is relatively new and not much code would break because of the missing backwards compatibility